lisa/analysis/latency.py

```diff
@@ -45,7 +45,7 @@ def df_latency_wakeup(self, task):
         DataFrame of a task's wakeup latencies
 
         :param task: The task's name or PID
-        :type task: int or str
+        :type task: int or str or tuple(int, str)
```

Comment:
Even though it's supposed to be equivalent, wouldn't it better to point to `TaskID` ?

Reply:
The intention was to let people pass actual tuples, not necessarily `TaskID` (although they can). Since that is an opaque value, it the only thing that matters is that `get_task_id(s)` knows how to handle it, and then always return a `TaskID` so the rest of the code only needs to know how to manipulate `TaskID`.